lib/driver.js

```diff
@@ -497,6 +508,10 @@ class IosDriver extends BaseDriver {
       }
       logger.debug('Waiting for initial webview');
       await this.navToInitialWebview();
+      // we don't need UIAutoClient and Instruments any more,
+      // should stop them to avoid 'Abnormal Instruments error' due to the command timeout.
+      await this.stopUIAutoClient();
```

Comment:
Hi @jlipps 

I didn't test on Simulator because I only support real device. But if we still use the instrument to launch Safari Browser may the bug still affect too. Because in long time (more than 1hour) the instrument doesn't get any command from uiAutoClient and may it got timeout.